js/flow/src/flow.ts

```diff
@@ -790,43 +397,38 @@ export class Flow<
         'Transfer-Encoding': 'chunked',
       });
       try {
-        const state = await this.runDirectly(input, {
+        const result = await this.invoke(input, {
           streamingCallback: ((chunk: z.infer<S>) => {
             res.write(JSON.stringify(chunk) + streamDelimiter);
           }) as S extends z.ZodVoid ? undefined : StreamingCallback<z.infer<S>>,
           auth,
         });
-        res.write(JSON.stringify(state.operation));
+        res.write({
+          result: result.result, // Need more results!!!!
```

Comment:
?

Reply:
`result: result.result`... joke?! I try...

Reply:
Oh I saw that and was like jeeze how many more are you expecting? Didn't get it was a meta-joke. :-)